### DISCUSSION TITLE
Where to get a list of settings.

### DISCUSSION THREAD

**Discussion by Dennisnuel** [community/community#20965] [Copilot Conversations]
Maybe I am dense, but where do I get a list of possible settings for the visual studio code plugin.

The documentation for settings is pretty sparse.
The documentation for "github.copilot.enable" does not explain what languages can be added and the syntax for declairing each language. I saw this page referenced in feature contributions https://code.visualstudio.com/docs/languages/identifiers. I figured these were the settings.
I am trying to use Copilot with PowerShell; it seems to work well after adding "powershell": true to the settings.
It's just that ... I am using PowerShell 5 to create a server setup script and need to use the default install of PowerShell on server 2019. Can I select the version of the language I am using?

There is no explanation for github.copilot.advanced
I have found a forum post saying that some of the open ai API settings work with the Visual Studio Code settings.
For example "temperature" = 20. There was also a setting for restricting the length of the results by symbol count.

Please can someone direct me to a more verbose explanation of the visual studio code plugin?

**Comment by fisforfaheem** [comment]
agree, i am lookign for it my self not found
